src/support/err.c

```diff
@@ -604,7 +604,7 @@ __wt_verbose_worker(WT_SESSION_IMPL *session, WT_VERBOSE_CATEGORY category, WT_V
     va_list ap;
 
     va_start(ap, fmt);
-    WT_IGNORE_RET(__eventv(session, true,
+    WT_IGNORE_RET(__eventv(session, level == WT_VERBOSE_ERROR ? false : true,
```

Comment:
@alisonfel, the change looks good to me but I was wondering if we could remove this argument and directly to the check on the level value inside `__eventv` ?

Reply:
As discussed, I've removed the `msg_event` parameter  09a9d26